.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+node_js:
+  - "0.12"
```

Comment:
We run everything below against all of these versions of node. A new virtual machine is spawned for each version to run the stuff below.